classes/controller/AdminController.php

```diff
@@ -2148,7 +2148,7 @@ protected function addPageHeaderToolBarModulesListButton()
         if (is_array($this->tab_modules_list['slider_list']) && count($this->tab_modules_list['slider_list'])) {
             $this->page_header_toolbar_btn['modules-list'] = array(
                 'href' => $this->getAdminModulesUrl(),
-                'desc' => $this->l('Recommended Modules and Services')
+                'desc' => $this->l('Recommended Modules')
```

Comment:
Although these strings are probably not used anymore, why do you change them?

Reply:
They are! @TristanLDD and @colinegin asked me to because it would make an overly long button otherwise

Reply:
Okay, marking as solved